examples/cli/discovery.md

```diff
@@ -1,11 +1,13 @@
 # Guiding Model Discovery
 
 This chapter of the `benchit` CLI tutorial is focused on how to guide the tool as it discovers models. You will learn things such as:
-- [How to run model discovery, without spending time on builds or benchmarking](#analyze-only)
-- [How to tell `benchit` where to look for the scripts that contain models](#search-directory)
-- [How to benchmark all the models in all the scripts in a directory](#benchmark-all-scripts)
-- [How to analyze the building blocks of a model](#maximum-analysis-depth)
-- [How to filter which models are passed to the build and benchmark operations](#filtering-model-hashes)
+- [Guiding Model Discovery](#guiding-model-discovery)
+- [Model Discovery Tutorials](#model-discovery-tutorials)
+  - [Analyze Only](#analyze-only)
+  - [Benchmark Multiple Scripts](#benchmark-multiple-scripts)
+    - [Maximum Analysis Depth](#maximum-analysis-depth)
+  - [Filtering Model Hashes](#filtering-model-hashes)
+- [Thanks!](#thanks)
```

Comment:
Please revert this change or merge with the original style. The purpose of the old table of contents was to explain to users why they would want to take any of the tutorials, by telling them what they would learn. A title like "maximum analysis depth" means nothing to a beginning user.

Reply:
VScode automatically updated that. Good catch.

Reply:
Hah your vscode is fancier than mine